README.md

```diff
@@ -71,6 +82,12 @@ Download a regional GeoJSON of hotspots detected by the MODIS satellite in a rec
 >>> data = nasa_wildfires.get_modis('usa-hawaii')
 ```
 
+Download a regional GeoJSON of hotspots detected by the MODIS satellite in a recent 7-day period.
+
+```python
+>>> data = nasa_wildfires.get_modis('usa-hawaii', '7d')
```

Comment:
Just to be as explicit as possible, let's use the kwargs here in this example. Like `time='7d'`